packages/react-network/src/server.ts

```diff
@@ -26,6 +26,11 @@ export function applyToContext<T extends Context>(
   Object.entries(cookies).forEach(([cookie, options]) => {
     const {value, ...cookieOptions} = options;
 
+    // only set cookie that had been changed
+    if (ctx.cookies.get(cookie) === value) {
```

Comment:
this might want to actually be `if (ctx.cookies.get(cookie) == value || ctx.cookies.get(cookie, options) == value)`, I'm not exactly sure how `options` interacts with `cookies.get`

Reply:
checked this out, options for `cookies.get` not the same as `cookies.set`
https://github.com/pillarjs/cookies#cookiesget-name--options--

so it should be
```
if (ctx.cookies.get(cookie) === value || ctx.cookies.get(cookie, {signed: true}) === value)
```